lib/saluki-components/src/transforms/aggregate/mod.rs

```diff
@@ -750,6 +751,32 @@ where
         // If we're dealing with a histogram, we calculate a configured set of aggregates/percentiles from it, and emit
         // them as individual metrics.
         MetricValues::Histogram(ref mut points) => {
+            // Convert histogram to distribution
+            if hist_config.copy_to_distribution() {
+                for (timestamp, histogram) in points.clone() {
+                    if timestamp.is_some() {
```

Comment:
Skipping if `timestamp` is `None` (when would this be?) since `SketchPoints` cannot use a default timestamp of `0`